src/components/container/SurveyAnswerChart/SurveyAnswerChart.tsx

```diff
@@ -53,11 +55,16 @@ export default function SurveyAnswerChart(props:SurveyAnswerChartProps) {
                         : intervalType === "6Month" ? add(intervalStart, { months: 6 }) :
                         intervalStart;
     const loadCurrentInterval = () => {
-        setCurrentData(null);
+        setSurveyAnswers(null);
         if (props.previewDataProvider) {
             props.previewDataProvider(intervalStart, intervalEnd)
             .then((data) => {
-                setCurrentData(processPages(data));
+                setSurveyAnswers(data);
+            });
+            return;
+        }else if(!!props.previewState){
+            getDefaultPreviewData(intervalStart, intervalEnd, props.series).then((data) => {
```

Comment:
@ajmenca Will we want to toggle when the series changes in vb?

Reply:
This is mostly following the pattern we've established in the DailyDataChart.  If there's a Preview Provider function, that is prioritized.  Otherwise if the previewState flag is set, we generate data in a pre-determined way.  In this case, that predetermined way takes into account how many Series you've passed in, which is controlled/set in View Builder.  So this all properly updates when that happens.